src/mono/wasm/debugger/DebuggerTestSuite/SteppingTests.cs

```diff
@@ -250,6 +250,38 @@ public async Task InspectLocalsInPreviousFramesDuringSteppingIn(bool use_cfo)
             );
         }
 
+        [Fact]
+        public async Task InspectNullStringBuilderLocalsDuringSteppingIn()
```

Comment:
What is special about `StringBuilder`? We have other tests for null object locals, IIRC.

Reply:
debugger handles `object` as `MONO_TYPE_OBJECT` and `StringBuilder` as `MONO_TYPE_CLASS`. They are 2 different code paths. We fixed both, we are testing both.